README.md

```
@@ -181,30 +161,28 @@
   db_data:
 ```
 
-Use with: `docker_compose` action: `deploy`
+Use with: "Deploy wordpress stack to production-1 with this compose file"
 
 ### Monitor Multiple Hosts
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)